packages/wrangler/src/__tests__/deploy.test.ts

```diff
@@ -255,6 +256,7 @@ describe("deploy", () => {
 
 			it("should not throw an error in non-TTY if 'CLOUDFLARE_API_TOKEN' & 'account_id' are in scope", async () => {
 				process.env = {
+					...process.env,
```

Comment:
Spreading `process.env` here is required, as otherwise `os.tmpdir()` fails on Windows. I'm not really sure how this worked previously with Jest